CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.4.0 (Unreleased)
+
+BUG FIXES:
+* resources/opennebula_virtual_network: Zero is a valid ID
```

Comment:
Please add a precisiopn saying it is for reservation_vnet you do the fix. Thanks